src/components/MonacoEditor/MonacoEditor.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import { spacing, colors } from '@ndla/core';
+
+import 'monaco-editor/esm/vs/editor/browser/controller/coreCommands';
+import 'monaco-editor/esm/vs/editor/browser/widget/codeEditorWidget';
+import 'monaco-editor/esm/vs/editor/contrib/bracketMatching/bracketMatching';
+import 'monaco-editor/esm/vs/editor/contrib/find/findController';
+import 'monaco-editor/esm/vs/editor/contrib/links/links';
+import 'monaco-editor/esm/vs/editor/contrib/suggest/suggestController';
+import 'monaco-editor/esm/vs/editor/contrib/wordHighlighter/wordHighlighter';
+import 'monaco-editor/esm/vs/editor/standalone/browser/quickOpen/quickCommand';
+import 'monaco-editor/esm/vs/editor/contrib/multicursor/multicursor';
+import 'monaco-editor/esm/vs/editor/contrib/linesOperations/linesOperations';
+import 'monaco-editor/esm/vs/editor/contrib/fontZoom/fontZoom';
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+
+// import * as monaco from 'monaco-editor';
```

Comment:
fjernes?

Reply:
It's nice to have for easy testing of all editor features. I'll add a comment.